docs/TEAM.md

```diff
@@ -34,7 +34,7 @@
 | |[Aravind Sivasailam](https://github.com/aravindsivasailam) | I'm a product engineer living in US with 2 years of experience in software testing and project management. New to software development and wish to improve in HTML, CSS and JS. |
 | |[Edwin Mancipe](https://github.com/efmg0325)|I am from Colombia. I have a bit of knowledge about HTML, CSS, Javascript, Python and SQL. I love to listeninng to Heavy Metal music. I like second life web browser games.|
 | |[Edori Atiri](https://github.com/EdoriAtiri) | From Nigeria. Began to learn Web Development this year. I hope to learn and contribute while I'm here.|
-
+| <img src="https://avatars2.githubusercontent.com/u/58055517?s=400&u=7fc1b9106782f52a5b563a7ae4ca277ea00cd9c3&v=4" width="225px" height="100px"> | [Fusen Ye](https://github.com/Yexiansen66) | I'm a beginner from China. I know HTML,CSS and JavaScript. |
```

Comment:
@Yexiansen66 Can you add your image in the `docs\images\team` directory and use the same here.

Reply:
I changed.But I have a new error: Docker build failed with exit code 1.What should I do?

Reply:
I changed.But I have a new error: Docker build failed with exit code 1.What should I do?

Reply:
We haven't use docker yet. May be you can ping me on discord to check this issue.